test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompaction_1_IT.java

```diff
@@ -450,6 +454,52 @@ public void testExternalCompactionDeadTServer() throws Exception {
     }
   }
 
+  @Test
+  public void testExternalCompactionWithFencedFiles() throws Exception {
+    String[] names = this.getUniqueNames(2);
+    try (AccumuloClient client =
+        Accumulo.newClient().from(getCluster().getClientProperties()).build()) {
+
+      String table1 = names[0];
+      createTable(client, table1, "cs1");
+
+      String table2 = names[1];
+      createTable(client, table2, "cs2");
+
+      writeData(client, table1);
+      writeData(client, table2);
+
```

Comment:
After writing data, could verify the table is as expected before fencing.

```suggestion
      verify(client, table1, 1, MAX_DATA);
      verify(client, table2, 1, MAX_DATA);
```